pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-inject-parameters.adoc

```diff
@@ -36,7 +35,7 @@ async fn request_filter(state: RequestState) -> RequestData<String>;
 async fn response_filter(state: ResponseState, RequestData(path): RequestData<String>);
 ----
 
-Instead of a function definition of:
+Instead of a function definition of defining the lambda as in the following example:
```

Comment:
Maybe I'd rephrase this to a more natural expression:

"Instead of using a lambda as parameter:"